simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/websocket/AbstractWebSocketClientInboxMessage.java

```diff
@@ -50,23 +50,23 @@ public int getSequenceId() {
         return Optional.empty();
     }
 
-    public final static class Connect extends WebSocketClientInboxMessage {
+    public static final class Connect extends AbstractWebSocketClientInboxMessage {
         @Override
         public Type getType() {
             return Type.CONNECT;
         }
     }
 
-    public final static class Disconnect extends WebSocketClientInboxMessage {
+    public static final class Disconnect extends AbstractWebSocketClientInboxMessage {
         @Override
         public Type getType() {
             return Type.DISCONNECT;
         }
     }
 
-    public final static class OcppMessage extends WebSocketClientInboxMessage {
+    public static final class OcppMessageAbstract extends AbstractWebSocketClientInboxMessage {
```

Comment:
Why OcppMessageAbstract has to be abstract? and even if it has to be abstract - I suppose that naming should be aligned with parent (abstract goes as a prefix)

Reply:
My bad, intellij automatically renamed also the inner class when i renamed the main class